numpy/lib/tests/test_io.py

```diff
@@ -1829,6 +1833,115 @@ def test_auto_dtype_largeint(self):
         assert_equal(test['f1'], 17179869184)
         assert_equal(test['f2'], 1024)
 
+
+class TestPathUsage(TestCase):
+    # Test that pathlib.Path can be used
+    @np.testing.dec.skipif(Path is None,
+                           "Cannot test without a pathlib.Path instance")
+    def test_loadtxt(self):
+        with temppath(suffix='.txt') as path:
+            path = Path(path)
+            a = np.array([[1.1, 2], [3, 4]])
+            np.savetxt(path, a)
+            x = np.loadtxt(path)
+            assert_array_equal(x, a)
+
+    @np.testing.dec.skipif(Path is None,
+                           "Cannot test without a pathlib.Path instance")
```

Comment:
Could shorten "Cannot test without a pathlib.Path instance" to "No pathlib.Path". Also below.